examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Can we not use special formatting for Merlin dataloader in the headline? We haven't done that in any other example. We do not use special formatting in the rest of the notebook.</p><p> </p><p>We do not use NVTabular in the notebook, so we should not confuse the reader in the beginning with it. We should remove it.</p><p> </p><p>The Overview should be something like:</p><p> </p><p>"Merlin dataloader is a library for constructing highly optimized dataloaders to accelerate training pipelines in TensorFlow (Keras) and PyTorch. In this example, we will provide a simple pipeline to train a MatrixFactorization Model in TensorFlow with Merlin dataloader based on the MovieLens dataset.</p><p> </p><p>The core features of Merlin dataloader:</p><p>- Accelerate pipelines by upto 10x compared to other dataloaders</p><p>- Handles larger than memory dataset by streaming data from disk</p><p>- Support for common data formats: CSV, Parquet, Avro</p><p>- Distributed training support</p><p> </p><p>Learning Objectives:</p><p>- Using Merlin dataloader to train a TensorFlow Keras Model</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>changes made</p>